scripts/.gitignore

```diff
@@ -0,0 +1 @@
+tmp*
```

Comment:
Please revert this file (if we need a gitignore we can add a complete one in a separate PR)